wordpress-mobile/gutenberg-mobile
# Issue/207 quotes design review

Fixes #207 

Related GB PR: https://github.com/WordPress/gutenberg/pull/15990

This PR addresses the design review made in #207. It changes the following:
 - Spaces quote and citation by 16px
 - Make the citation font size to be 14px

Here is the end result:

![Simulator Screen Shot - iPhone Xʀ - 2019-06-04 at 22 36 29](https://user-images.githubusercontent.com/651601/58915872-e6587600-8719-11e9-8031-ce59277e5750.png)

To test:
 - Run the demo app
 - Check that the quotes styles is correct.

Update release notes:
